### ISSUE TITLE
Migrate blog from blog.pact.io to pact.io/blog

### ISSUE THREAD

**Issue by bethesque** [pact-foundation/docs.pact.io#54]

**Comment by Harshith2255** [comment]
Hi
Can I help?

**Comment by bethesque** [comment]
Thank you! Can you PM me your email address on Slack, and I'll invite you to the blog post admin panel. Given there are only 9 posts, I think just copying the markdown manually will be the easiest approach.

**Comment by bethesque** [comment]
I've enabled the blog, which you will find here https://docs.pact.io/blog/

You'll be able to see from [this commit](https://github.com/pact-foundation/docs.pact.io/commit/b64f47b3a96f2475f24a02c9482d3b149b58cdf4) where to put the posts.

I just copied the raw markdown from the Ghost admin by clicking on the edit icon for the post when signed in as admin.

![Screen Shot 2021-10-01 at 4 49 31 pm](https://user-images.githubusercontent.com/446228/135577637-c2cd4757-9e4c-44b0-8f1b-2af7be6b09cb.png)
 
To get the published URL of the post, click on the Settings cog, then "view post". Once they're all moved, we'll put in a redirect from the old site to the new one.
